pkg/interpreter/simple/simple.go

```diff
@@ -0,0 +1,71 @@
+package simple
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/caicloud/ciao/pkg/types"
+)
+
+type Interpreter struct {
```

Comment:
note for improvement (future PR)

This is the core API of the kernel but it's nested too deeply, I would imagine this to be easily discoverable. The specific commands should be defined somewhere else and preferably versioned, and interpreter just imports the commands?  This is also easier for us to implement features like list all available commands, etc